src/main/java/cz/sparko/boxitory/service/filesystem/FilesystemDescriptionProvider.java

```diff
@@ -83,9 +97,21 @@ private DescriptionLine parseLine(String line) {
         if (splittedLine.length != 2) {
             return null;
         }
+
+        if (versionAsTimestamp) {
```

Comment:
can you please move this logic to the ```getDescription``` method? That will probably need some refactoring as there is already a lot of logic in that method.
This method is named ```parseLine``` and this logic do some modification to that. It seems like sideeffect.